rdt/transformers/numerical.py

```diff
@@ -58,6 +71,10 @@ def fit(self, data):
         self._dtype = self.dtype or data.dtype
         self.null_transformer = NullTransformer(self.nan, self.null_column, copy=True)
         self.null_transformer.fit(data)
+        if self.min_value == 'auto':
+            self.min_value = min(data)
```

Comment:
Maybe we should store the min and max values in a different attribute (`_min_value` and `_max_value`) to avoid overwriting the user input, so a single instance can be fitted multiple times on different data with consistent behavior.

Reply:
I added those variables but I get the following after running `pylint rdt --rcfile=setup.cfg`: 

`rdt\transformers\numerical.py:14:0: R0902: Too many instance attributes (9/7) (too-many-instance-attributes)`